### PR TITLE
[#1731] Updating the Eclipse Support webpage.

### PR DESCRIPTION
**Pull request by miklossy**
- Add Mark Occurrences and Find References section.
- Improve Folding section to break the long lines of the embedded
code snippets properly.

Signed-off-by: Tamas Miklossy <miklossy@itemis.de>

### CODE REVIEW

File: xtext-website/documentation/310_eclipse_support.md [hunk 1]
```diff
@@ -604,6 +613,36 @@ public class MyDslUiModule extends AbstractMyDslUiModule {
 }
 ```
 
+## Mark Occurrences {#maroccurrences}
+Xtext-based editors are able to highlight all occurrences of a certain element in the opened DSL file. Once the user selects an element via double click while the `Toggle Mark Occurrences` button is enabled, all occurrences are highlighted with corresponding markers on the right side of the editor.
```
Comment (cdietrich) [inline xtext-website/documentation/310_eclipse_support.md:617]: double click is new to me. thought a single one is sufficient

File: xtext-website/documentation/310_eclipse_support.md [hunk 2]
```diff
@@ -604,6 +613,36 @@ public class MyDslUiModule extends AbstractMyDslUiModule {
...
+	
+	public Class<? extends IOccurrenceComputer> bindIOccurrenceComputer() {
+		return MyDslOccurrenceComputer.class;
+	}
+
+}
+```
+
+## Find References {#findreferences}
+Xtext-based editors are able to locate all references in the entire workspace where a certain element is referred to. Invoking the `Find References` context menu or using the keyboard shortcut `Ctlr+Shift+G` on the selected element lists all the references in the `Search` view.
```
Comment (cdietrich) [inline xtext-website/documentation/310_eclipse_support.md:640]: Cmd+Shift+G on Mac

File: xtext-website/documentation/310_eclipse_support.md (hunk 1 above)
Comment (miklossy) [reply xtext-website/documentation/310_eclipse_support.md]: Fixed.

File: xtext-website/documentation/310_eclipse_support.md (hunk 2 above)
Comment (miklossy) [reply xtext-website/documentation/310_eclipse_support.md]: Fixed.

**Pull request merged**